lib/homepage.rb

```diff
@@ -0,0 +1,35 @@
+class Homepage
+
+  attr_reader :currently_viewed_menu
+
+  def initialize
+    @restaurant_menus = []
+    @restaurants = []
+    @currently_viewed_menu = nil
+    @order = nil
+  end
+
+  def display_restaurants
+    return @restaurants.join("\n") unless @restaurants.empty?
+    find_restaurants
+    @restaurant_menus.each_with_index do |name, index|
+      @restaurants << "#{index + 1}: #{name.chomp('_menu.csv').gsub("./lib/", "")}"
+    end
+    @restaurants.join("\n")
+  end
+
+  def see_restaurant_menu(restaurant)
+    @currently_viewed_menu = Menu.new(restaurant)
```

Comment:
This is a nice bit of abstraction (think that's the word) - didn't occur to me at all